guides/development-workflow.md

```diff
@@ -401,27 +414,87 @@ Upgrade widget to v1.2.1
 
 ## Code review
 
-We believe that code reviews are important. They help us to learn from each
-other, to maintain consistency in the code we write, and to ensure we push
-well-reasoned and bug-free code to Production. When performing a code review, we
-make sure that:
-
-- we cannot see any code that could give rise to a security vulnerability
-- the story has been implemented clearly and maintainably
-- the code contains well-written commits that help us to understand the changes
-- the code follows code style guides
-- the [changelog](#tracking-changes) has been updated is the change was
-  significant
+Code reviews help us ship better quality code and provide an opportunity to
+learn from each other.
+
+### How to begin
+
+To help review code changes you can
+[enable split review within GitHub](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-comparing-branches-in-pull-requests#diff-view-options).
```

Comment:
Can we be clearer that this pointing out an option that you may find helpful, rather than something you should definitely do? Or did you intend for it to be a recommendation?